content/1.7/back-office/shipping/carriers/add-edit.md

```diff
@@ -121,5 +121,4 @@ WIP
 
 ## Behaviors / User stories
 
-WIP
-
+When you edit a carrier a new carrier is created. a link is saved to the original id with the id_reference.
```

Comment:
```suggestion
When you edit a carrier, a new carrier is created and the carrier id changes. A link is saved to the original id with the id_reference.
```
Hello @okom3pom and thank you for this PR
Could you tell me more about the link saved: where is it saved?